python/tank/platform/engine.py

```diff
@@ -123,7 +122,7 @@ def __init__(self, tk, context, engine_instance_name, env):
         
         # create invoker to allow execution of functions on the
         # main thread:
-        self._invoker = self.__create_main_thread_invoker()
+        self.__create_invokers()
```

Comment:
Just a personal opinion, so take it with a pinch of salt - i always liked initializing member varialbles in the constructor if possible, it gives a reader of the code a good sense of what member states and variables exist simply by looking at the constructor. So I think I would have done

```
self._invoker = None
self._async_invoker = None
self.__create_invokers()
```

But more of a personal opinion than anything else...